bokeh/server/server.py

```diff
@@ -34,17 +57,22 @@ def __init__(self, applications, **kwargs):
 
         tornado_kwargs = { key: kwargs[key] for key in ['io_loop',
                                                         'extra_patterns',
-                                                        'keep_alive_milliseconds']
+                                                        'keep_alive_milliseconds'
+                                                        'hosts']
                            if key in kwargs }
 
-        self._tornado = BokehTornado(self._applications, **tornado_kwargs)
-        self._http = HTTPServer(self._tornado)
         self._port = DEFAULT_SERVER_PORT
         if 'port' in kwargs:
             self._port = kwargs['port']
+
+        tornado_kwargs['hosts'] = _create_hosts_whitelist(kwargs.get('host', None), self._port)
```

Comment:
much more insidious error too --- no comma, the last two names were concatenated